Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs

```
@@ -49,6 +49,11 @@ public static partial class MXRAndroidUtils {
             Plugin.Call<bool>("launchIntentAction", intentAction);
         }
 
+        public static void OpenURLWithOculusBrowser(string url) {
+            if (Plugin != null)
```

If the Plugin is null could we throw an exception / log ?